src/main/ts/tasks/GruntRollup.ts

```diff
@@ -110,7 +110,7 @@ export const task = (grunt) => {
             grunt.file.write(file.dest, output.code);
           }
         } else {
-          grunt.file.write(file.dest, output.source);
+          grunt.file.write(file.dest, String(output.source));
```

Comment:
It's probably not worth it, but this could be `output.source.toString()` like the others above because both types (`string` and `Uint8Array`) support that method ¯\_(ツ)_/¯ 

```suggestion
          grunt.file.write(file.dest, output.source.toString());
```